web_monitoring/differs.py

```diff
@@ -65,13 +74,14 @@ def html_text_diff(a_text, b_text):
     ------
     >>> html_text_diff('<p>Deleted</p><p>Unchanged</p>',
     ...                '<p>Added</p><p>Unchanged</p>')
-    [(0, '<p>'), (-1, 'Delet'), (1, 'Add'), (0, 'ed</p><p>Unchanged</p>')]
+    [[-1, 'Delet'], [1, 'Add'], [0, 'ed Unchanged']]
```

Comment:
It isn't a typo @suchthis, this is the result from the algorithm. If you see the two strings, `Deleted Unchanged` and `Added Unchanged`, you'll see that only the part before `ed` in `Deleted` has changed, the remainder of the string doesn't change. The function returns what part of the string has been removed with the tag -1, addition with +1, and no change with 0.